### ISSUE TITLE
magicitem command not working

### ISSUE THREAD

**Issue by Starisho** [RandyKnapp/ValheimMods#429]
Latest version of mod and game, I was trying to mess around with a fresh character and see if it was safe to play on my old save from before Mistalnds. Upon using the magicitem command, it seems like the game recognizes the command and when it works, but despite outputting to the console the random item spawned such as "Succubus Aetgir" no item actually drops. I cannot get any legendary items to drop via this command when previously it used to.

**Comment by Starisho** [comment]
the magicitemset command does work however

**Comment by Devestatio** [comment]
The command to spawn items is magicitem <rarity> <itemtype> <amount>

So you would type magicitem legendary Aetgir 1 for that to work. What your typing is a specific name that was randomly rolled for the item during creation, to my knowledge that has never worked.

**Comment by Starisho** [comment]
You misunderstand, I did the command just as you said. I typed “magicitem legendary 1” to roll a random item. I’ve used it before and I’m aware how it works haha, the name I specifically gave was an example of that random item you mentioned. So I was trying to roll a random legendary with the command, it gave an aetgir as the random weapon. I know the command worked because it told me which weapon was rolled. Like the console spat out a specific weapon name when I did not include any such specificity in my command. However, the only issue is that anything I roll with magicitem does not actually drop in the world. For example typing “magicitem legendary MaceIron 1” runs with no errors, tells me it was successfully spawned, but the item was not actually spawned into the world. So all I know is that it successfully randomizes an item, but it is nowhere to be seen and therefore cannot be picked up. I hope this makes sense and clears things up :)

TL;DR I used the command “magicitem legendary 1” I did not specify any weapon, console printed out to me what the random item that was rolled was. I did not specify it myself. Therefore, command worked since the randomized worked. The issue was that any and all items spawned via magicitem command never drop in the world.

**Comment by Vapok** [comment]
This has been fixed with 0.9.2
